contracts/src/token/erc1155/mod.rs

```diff
@@ -179,14 +182,14 @@ impl MethodError for Error {
     }
 }
 
-sol_storage! {
-    /// State of an [`Erc1155`] token.
-    pub struct Erc1155 {
-        /// Maps users to balances.
-        mapping(uint256 => mapping(address => uint256)) _balances;
-        /// Maps owners to a mapping of operator approvals.
-        mapping(address => mapping(address => bool)) _operator_approvals;
-    }
+/// State of an [`Erc1155`] token.
+#[storage]
+pub struct Erc1155 {
+    /// Maps users to balances.
+    pub _balances: StorageMap<U256, StorageMap<Address, StorageU256>>,
```

Comment:
We will need the struct to be public to be able to inherit and use the contract, but fields should probably be private

Reply:
I wanted to make it private everywhere. But some of our tests assuming, that it should be public. Decided to create an [issue](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/447) for that. Since by the end it will be a breaking change

Reply:
this pr is about code coverage

Reply:
Fair enough!